cis/system/ksp-audit-cis-mongodb-5.1.yaml

```diff
@@ -0,0 +1,23 @@
+# KubeArmor is an open source software that enables you to protect your cloud workload at run-time.
+# To learn more about KubeArmor visit: 
+# https://www.accuknox.com/kubearmor/ 
+
+apiVersion: security.kubearmor.com/v1
+kind: KubeArmorPolicy
+metadata:
+  name: ksp-audit-cis-mongodb-5.1
+  namespace: default # Change your namespace
+spec:
+  tags: ["CIS","MONGODB"]
```

Comment:
please add rule number to the tags @harshaccuknox

Reply:
Added, please check.